tests/unit/test-firefox/test.firefox.js

```diff
@@ -545,6 +646,136 @@ describe('firefox', () => {
 
   });
 
+  describe('defaultCreateProfileFinder', () => {
+
+    it('creates a finder', async () => {
+      const FxProfile = {
+        Finder: sinon.spy(() => () => Promise.resolve()),
```

Comment:
`Finder` is a constructor function and so it seems that its spy should be something like:

```js
const FxProfile = {
  Finder: sinon.spy(function () { 
    return {}; 
  }),
};
```